js/pages/myToDoList.js

```diff
@@ -754,8 +754,23 @@ const setModuleAttributes = (data, modules, collections) => {
     }
 
     if(data[fieldMapping.menstrualSurveyEligible] === 353358909) {
-        modules['Menstrual Cycle'].enabled = true;
 
+        // Survey will become available on dashboard 120 hours (5 days) after completion of bio survey if participant is menstrual survey eligible
+        const date = new Date();
+        date.setHours(date.getHours() - 120);
```

Comment:
no need to create all of this logic, it all exists below... just implement initial five day gap into what already exists

Reply:
Solved